### PR TITLE
02 Fx/ 00 TypeScript

### PR DESCRIPTION
**Pull request by jacobovirtual**
#60 All is correct, only I change one picture that not correspond to process description.

### CODE REVIEW

**Comment by nasdan** [comment]
Hi @jacobovirtual, about the image, it's correct because it's representing that when we use `import * as $ from 'jquery'` we have intellisense for jquery library

**Comment by jacobovirtual** [comment]
I think the highlighted CSS line can cause confusion. That is why I changed the image.
Because at the moment I'm not configuring intellisense for jquery.

**Comment by nasdan** [comment]
Before image, we have configured intellisense for jquery:

- `npm install @types/jquery --save-dev`
- And `import * as $ from 'jquery';`

**Comment by nasdan** [comment]
There are conflicts with master, could you solve, please?

I think you have to pull the last version of master to your repository, because in this issue #62 I solved Readme links.

**Comment by jacobovirtual** [comment]
I'm sorry, I'm forgot launch a 'git pull lemoncode master' command over my branch.

File: 02 Fx/00 TypeScript/README.md [hunk 1]
```diff
@@ -97,6 +97,11 @@ document.write(messageToDisplay);
 
 ```
 
+<<<<<<< HEAD
```
Comment (nasdan) [inline 02 Fx/00 TypeScript/README.md]: Remove conflicts stuff

File: 02 Fx/00 TypeScript/README.md [hunk 2]
```diff
@@ -97,6 +97,11 @@ document.write(messageToDisplay);
...
+- Here we can see the result of instalation the jquery typings
```
Comment (nasdan) [inline 02 Fx/00 TypeScript/README.md]: Typo `installing`

**Comment by jacobovirtual** [comment]
I don't believe, I'm forgot save the file.
-1 to me

File: 02 Fx/00 TypeScript/README.md [hunk 3]
```diff
@@ -97,6 +97,8 @@ document.write(messageToDisplay);
 
 ```
 
+- Here we can see the result of instalation the jquery typings
```
Comment (nasdan) [inline 02 Fx/00 TypeScript/README.md:100]: I think is better to say: `result by installing`.
What do you think?

**Comment by nasdan** [comment]
Hi @jacobovirtual, do you have pending push the changes or something like that?

**Pull request closed without merging**